dotcom-rendering/src/web/layouts/LiveLayout.tsx

```diff
@@ -662,12 +658,10 @@ export const LiveLayout = ({ CAPI, NAV, format, palette }: Props) => {
 							<LiveGridSport>
 								<GridItem area="matchtabs" element="aside">
 									<div css={maxWidth}>
-										{CAPI.matchUrl && showMatchTabs && (
-											<Placeholder
-												rootId="match-tabs"
-												height={40}
-											/>
-										)}
+										<Placeholder
+											rootId="match-tabs"
+											height={40}
+										/>
```

Comment:
We’re already in the truthy block of `CAPI.matchUrl`.

Reply:
🤦 great spot!

Reply:
That’s @joecowton1 😉!